### ISSUE TITLE
Possible "bug" with multiple erasers/deleters

### ISSUE THREAD

**Issue by medicatedshawnie** [sigma144/witness-randomizer#6]
I'm honestly unsure whether this is a bug since the game acts unusually when there are multiple deleters, but I thought you should know that the image here is considered a valid solution in-game, but online solvers (including the one that came with the randomizer) do not consider it a valid solution. The solvers, in fact, did not find any valid solutions. EDIT: I didn't mention that when solved the sun and both of the deleters all fade away.
![SOLVED](https://user-images.githubusercontent.com/60412657/73316784-18b36800-4202-11ea-9639-b9d5cee8c7d9.PNG)

**Comment by sigma144** [comment]
Yeah, that's how the game treats multiple deleters. Kind of weird, huh? Essentially, each deleter picks a symbol to erase independently, so it's possible that they choose the same symbol. I actually made this puzzle specifically to showcase this behavior. Because of it, the game will also not consistently handle solutions that require two deleters to erase a specific combination of 2 symbols in their group. This is why every puzzle after this (with one exception) forces the deleters to be separated into different groups. The included solver doesn't handle this because I didn't know about this behavior when I wrote the solver several years ago.

TL;DR: This is intended behavior, but thanks for verifying anyway!
